pkg/retriever/retriever_test.go

```diff
@@ -166,38 +166,38 @@ func setupAllEvents(t *testing.T, contracts *cutils.AllTestContracts) int {
 func TestRetrieveMethod(t *testing.T) {
 	contracts, retriever, _ := setupTestRetriever(t)
 
-	numEvents := setupAllEvents(t, contracts)
-	err := retriever.Retrieve()
+	_ = setupAllEvents(t, contracts)
+	err := retriever.Retrieve(false)
 	if err != nil {
 		t.Errorf("Error retrieving events: %v", err)
 	}
 	pastEvents := retriever.PastEvents
-	if len(pastEvents) != numEvents {
-		t.Fatalf("Should have collected %v events but collected %v", numEvents, len(pastEvents))
+	if len(pastEvents) != 7 {
```

Comment:
could you define 7 as a const so it doesn't seem like as much of a magic number?

Reply:
sure, but this is for test code.